data_validation/__main__.py

```diff
@@ -184,16 +202,15 @@ def build_config_from_args(args, config_manager):
             config_manager.append_comparison_fields(
                 config_manager.build_config_comparison_fields(comparison_fields)
             )
-            if args.hash != "*":
+            if args.hash != "*" or args.concat != "*":
                 config_manager.append_dependent_aliases(comparison_fields)
 
     if args.primary_keys is not None:
         primary_keys = cli_tools.get_arg_list(args.primary_keys)
         config_manager.append_primary_keys(
             config_manager.build_column_configs(primary_keys)
         )
-        if args.hash != "*":
```

Comment:
Shouldn't we do the same thing here i.e `if args.hash != "*" or args.concat != "*"`?
Since this way if the user wants to hash/concat all fields, we won't populate the dependent aliases at all since we don't need it